Tabloid/Models/Post.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace Tabloid.Models
```

Comment:
Do any of these have a max length you could account for?

Reply:
Added MaxLength to Title and ImageLocation properties.